synapse/handlers/deactivate_account.py

```diff
@@ -96,6 +96,10 @@ async def deactivate_account(
                 403, "Deactivation of this user is forbidden", Codes.FORBIDDEN
             )
 
+        logger.info(
+            f"{requester.user} requested deactivation of {user_id} erase_data={erase_data} id_server={id_server}"
```

Comment:
We shouldn't use f-strings for log lines

```suggestion
            "%s requested deactivation of %s erase_data=%s id_server=%s", requester.user, user_id, erase_data, id_server
```